### PR TITLE
[full-ci] Admin settings add group members panel

### PR DESCRIPTION
**Pull request by lookacat**
## Description
We've added the members panel to the groups page in admin settings.
For more details: https://github.com/owncloud/web/issues/8596

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8596

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: Hmmm I'm honestly not the biggest fan of combining space- and group-members in one and the same panel. Especially when we extend the functionality in the future to search/filter by specific attributes or make it possible to add/remove members.

Comment (JanAckermann) [reply packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: Yes, I was about to write the same. We did this already with the Quota Modal which looks super cluttery.

Comment (lookacat) [reply packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: so some code duplication is better in this case, right? would be fine with me too

Comment (JammingBen) [reply packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: It's always a fine line between making components generic & reusable and duplicating code. Here I'd say a separate component with a bit of duplication is better. I suspect at least the group member panel will be expanded by some edit functionality in the future.

Comment (lookacat) [reply packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: done

File: packages/web-app-admin-settings/src/components/Groups/SideBar/MembersPanel.vue [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import MembersRoleSection from '../../Groups/SideBar/MembersRoleSection.vue'
+import Fuse from 'fuse.js'
+import Mark from 'mark.js'
+import { Group } from 'web-client/src/generated'
+
+export default defineComponent({
+  name: 'GroupsMembersPanel',
+  components: { MembersRoleSection },
+  setup() {
+    const group = inject<Group>('group')
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Groups/SideBar/MembersPanel.vue:30]: I noticed recently that most of our `inject` types are wrong. When we pass them as `computed` or as `ref`, they are a `Ref`. 

```suggestion
    const group = inject<Ref<Group>>('group')
```

File: packages/web-app-admin-settings/src/components/Groups/SideBar/MembersRoleSection.vue [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<template>
+  <ul class="oc-list">
+    <li
+      v-for="(member, index) in groupMembers"
+      :key="index"
+      class="oc-flex oc-flex-middle oc-mb-s"
+      data-testid="space-members-list"
```
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Groups/SideBar/MembersRoleSection.vue:7]:
```suggestion
      data-testid="group-members-list"
```

File: packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue:1]: The changes in this file can be undone, no?

File: packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersRoleSection.vue
Comment (JammingBen) [inline packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersRoleSection.vue:1]: The changes in this file can be undone, no?

**Comment by JammingBen** [comment]
Code looks good to me! But searching gives me weird results 🤔 Does it happen for you as well @lookacat ?

image

**Comment by lookacat** [comment]
no actually not :/ what did you do?

**Comment by lookacat** [comment]
should now work fine

File: packages/web-app-admin-settings/tests/unit/components/Groups/SideBar/MembersPanel.spec.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import MembersPanel from '../../../../../src/components/Groups/SideBar/MembersPanel.vue'
```
Comment (kulmann) [inline packages/web-app-admin-settings/tests/unit/components/Groups/SideBar/MembersPanel.spec.ts:1]: the import looks wrong

**Review by kulmann — APPROVED** [review]
nice 👍

**Comment by kulmann** [comment]
> As discussed in chat: Highlighting the filter term does not work yet :(

Oh true... and now found that having marie in a group e.g. filter term `Curie` doesn't find a result o_O When I search for `Marie` the highlighting works (but swallows the blank), searching for `Mari` nothing is highlighted.

**Review by JammingBen — APPROVED** [review]
Works now 👍 

On a side note: the Fuse search in general is SUPER buggy though. I constantly get no results where there should definitely be some. Not only here, but basically everywhere we use it.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34908/23/1
:boom: The oC10SharingAccept tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34908/71/1
:boom: The oC10IntegrationApp2 tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34908/26/1
:boom: The oC10SharingIntGroups tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34908/25/1
:boom: The oC10Sharing2 tests pipeline failed. The build has been cancelled.

**Pull request merged**